static/js/components/tiles/line_tile.tsx

```diff
@@ -45,6 +46,10 @@ export interface LineTilePropType {
   className?: string;
   // colors to use
   colors?: string[];
+  // List of place DCIDs to plot, instead of enclosedPlaceType in place
+  comparisonPlaces?: string[];
```

Comment:
wonder if it would make more sense for comparison places to be places that we plot in addition to the main place? To be consistent with the behavior in other tiles (e.g., bar)

Reply:
I like the suggestion of maintaining consistency, but I think the current behavior in other tiles is to have comparisonPlaces to be _instead of_ the main place.

It looks like comparisonPlaces is only also used in the bar tile. The current data fetch doesn't get data for the main place if comparisonPlaces is set. See: https://github.com/datacommonsorg/website/blob/537b332b85959480abe982a10e7e564464140924/static/js/components/tiles/bar_tile.tsx#L172-L185

Based on how our TileConfig is set though, it looks like the intention is to have comparisonPlaces be in _addition to _the main place.

Should I update both the line tile and bar tile to plot main place in addition to comparisonPlaces? Or should match the line tile with the current behavior of the bar tile (either main place, or comparisonPlaces, but not both)?

Reply:
@dwnoble @chejennifer

Reply:
Going with matching comparisonPlaces with how it works in the bar chart. If requested, I'll update the behavior in a followup PR.